Prace_domowe/Praca_domowa2/Agata_Kaczmarek/Kaczmarek_Agata_hw2.ipynb

```diff
@@ -0,0 +1,1016 @@
+{
```

Comment:
<p>Nie do końca o to chodziło. Chodziło o porównanie danych imputowanych z oryginalnymi, a Ty porównałaś wszystkie dane (w tym oryginalne) z oryginalnymi.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>